### ISSUE TITLE
How do I install Blue Shift and Opposing Force?

### ISSUE THREAD

**Issue by jeffyjeffy1023** [masterfeizz/Xash3DS#30]
I wanna play the HL1 expansions on my 3DS but I'm not sure which files go where

**Comment by MixedVictor** [comment]
yes i am replying this 3 months later, just copy the Blue Shift/Opposing Force folder data folder (NOT THE DIRECTORY THAT CONTAINS THE EXECUTABLE THAT IS hl.exe) to the Xash3D folder in the sd card root directory, same can be applied to Counter Strike (cstrike) and other half life mods (For example Half-Life: Expanded Asernal, hl_expanded_asernal)

**Comment by MixedVictor** [comment]
also do not forget to put extras.pak inside them

**Comment by Foneet** [comment]
> yes i am replying this 3 months later, just copy the Blue Shift/Opposing Force folder data folder (NOT THE DIRECTORY THAT CONTAINS THE EXECUTABLE THAT IS hl.exe) to the Xash3D folder in the sd card root directory, same can be applied to Counter Strike (cstrike) and other half life mods (For example Half-Life: Expanded Asernal, hl_expanded_asernal)

i copied gearbox and bshift folders, added extras.pak in them, then in hl1 menu i go to custom game, activating opposing force  sends me to the home menu and blue shift crashes the console, what am i doing wrong?

**Comment by MixedVictor** [comment]
> > yes i am replying this 3 months later, just copy the Blue Shift/Opposing Force folder data folder (NOT THE DIRECTORY THAT CONTAINS THE EXECUTABLE THAT IS hl.exe) to the Xash3D folder in the sd card root directory, same can be applied to Counter Strike (cstrike) and other half life mods (For example Half-Life: Expanded Asernal, hl_expanded_asernal)
> 
> i copied gearbox and bshift folders, added extras.pak in them, then in hl1 menu i go to custom game, activating opposing force sends me to the home menu and blue shift crashes the console, what am i doing wrong?

i don't know it's been so long since i posted the comment lol, i may try installing later.

**Comment by thepwrtank18** [comment]
Same issue. Anything that isn't HL stops working.
